host/3.0/buster/amd64/java/java11/java11-slim.Dockerfile

```diff
@@ -19,6 +20,10 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     EXTENSION_BUNDLE_FILENAME=Microsoft.Azure.Functions.ExtensionBundle.1.8.1_linux-x64.zip && \
     apt-get update && \
     apt-get install -y gnupg wget unzip && \
+    wget https://aka.ms/download-jdk/microsoft-jdk-${JAVA_VERSION}-linux-x64.tar.gz && \
```

Comment:
Please move this to the top of the RUN command. Just so that it is not in the middle of ext bundle pulls and therefore easier to read.

Reply:
Apply this comment and others to all places that have identical code.

Reply:
updated as suggested.

Reply:
The build job failed because it cannot find wget, in this case should I keep it at the old places right after wget is installed.